lib/ach_client/version.rb

```diff
@@ -1,4 +1,4 @@
 module AchClient
   # Increment this when changes are published
-  VERSION = '0.4.0'
+  VERSION = '0.5.0'
```

Comment:
Meh I would say that this is a minor version change because it is backwards-compatible for public consumption. The only thing that would break is the creation of response objects, which only happens internally. From a public consumers point of view, they just get an extra field in their response objects from now on.
Also semvar states that major version 0 is for initial development, and there is no stable public api. I think we can commit to the public api once everything is running ok in our prod env (for polling of ACH's in addition to sending them which we already have now)
http://semver.org/